cmd/exporter.go

```diff
@@ -21,6 +22,12 @@ const (
 )
 
 func init() {
+	exporterCmd.Flags().StringP(providerKey, "p", provider.Ember, "Provider of carbon intensity data")
+	exporterCmd.Flags().StringP(regionKey, "r", "", "Region code for provider")
+
+	viper.BindPFlag(providerKey, exporterCmd.Flags().Lookup(providerKey))
+	viper.BindPFlag(regionKey, exporterCmd.Flags().Lookup(regionKey))
```

Comment:
Before provider and region were global flags but we don't need them for `provider list`.

So this adds them to the exporter command.